setup.py

```diff
@@ -212,17 +246,35 @@ class BuiltFile(_BaseExtension):
     `ext_modules`.
     """
 
-    def __init__(self, src: str, dst: str):
+    def __init__(
+        self,
+        src_path: str,
```

Comment:
Please call this `src_dir` since it's specifically a directory. `src_path` sounds like it could still refer to the path to the source file.